ui/raidboss/data/05-shb/dungeon/anamnesis_anyder.js

```diff
@@ -147,7 +147,7 @@
         'trench phuabo': 'Tiefsee-Phuabo',
         'trench danbania': 'Tiefsee-Danbania',
         'trench anemone': 'Tiefsee-Anemone',
-        'sinister bubble': 'Finster(?:e|er|es|en) Blase',
+        'Sinister Bubble': 'Finster(?:e|er|es|en) Blase',
```

Comment:
I think these are all case insensitive.  It's probably not worth changing the case on any keys.  I mean, this is fine as-is, but just saying for the future.

Reply:
i just change this as they were reported by the python script for checking missing sync/text